tslint.json

```diff
@@ -11,12 +12,14 @@
     "curly": true,
     "eofline": true,
     "forin": true,
+    "import-blacklist": [false, "rxjs"],
```

Comment:
This should also be set back to true.  But it requires changing every single `from 'rxjs'` import.  And figuring out exactly what operators we need.  Apparently does trim down the size of that lib quite a bit, though.